File: Modul 3/NO 5/PRAK205-2210817310016-Riyo Aurora Guison.py
import math
A,B = input(). split()
A =int(A)
B =int(B)
C =math.sqrt(B**2 - A**2)
T = A
K = A + B + C
L = C*T/2
print("Alas =",round(C),"cm")
print("Keliling =",round(K),"cm")
print("Luas =",round(L),"cm^2")

a =int(input())
b =int(input())
c =math.sqrt(b**2 - a**2)
t = a
k = a + b + c
l = c*t/2
print("Alas =",round(c),"cm")
print("Keliling =",round(k),"cm")
print("Luas =",round(l),"cm^2")

File: Modul 3/NO 5/PRAK205-2210817310016-Riyo Aurora Guison.c
#include <stdio.h>
#include<math.h>
int main ()
{
int A,B,hasil1,hasil2,hasil3;
scanf("%d",&A);
scanf("%d",&B); 
hasil1 =sqrt(B*B - A*A);
printf("Alas  = %d cm\n",hasil1);
printf("Tinggi = %d cm\n",A);
hasil2 = A + B + hasil1;
printf("Keliling = %d cm\n",hasil2);
hasil3 = hasil1*A/2;
printf("Luas = %d cm^2\n",hasil3);
}